apps/webapp/app/routes/_app.orgs.$organizationSlug.projects.v3.$projectParam.test/route.tsx

```diff
@@ -210,12 +210,10 @@ function TaskSelector({
       )}
       <Table>
         <TableHeader>
-          <TableRow>
-            <TableHeaderCell className="pl-3" colSpan={2}>
-              Task
-            </TableHeaderCell>
-            <TableHeaderCell className="px-2">File path</TableHeaderCell>
-          </TableRow>
+          <TableHeaderCell className="pl-3" colSpan={2}>
+            Task
+          </TableHeaderCell>
+          <TableHeaderCell className="px-2">File path</TableHeaderCell>
         </TableHeader>
```

Comment:
"You must place `<tr>` elements inside `<thead>` to contain your `<th>` cells. The correct structure is: `<thead><tr><th>Header Cell</th></tr></thead>`. [1](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/thead) [2](https://www.w3.org/TR/html401/struct/tables.html)"